src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -78,9 +72,14 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
 
   def /(that: DataWord): DataWord = DataWord(this.n / that.n)
 
+  def **(that: DataWord): DataWord = DataWord(this.n.modPow(that.n, Modulus))
+
   def compare(that: DataWord): Int = this.n.compare(that.n)
 
-  def intValue: Int = n.intValue
+  /**
+    * @return an Int with MSB=0, thus a value in range [0, Int.MaxValue]
+    */
+  def intValue: Int = n.intValue & Int.MaxValue
```

Comment:
Let's keep it as it is for now. We'll return to it in a follow-up PR when we'll have to implement division on both signed and unsigned ints.